src/js/components/projects/createProjectModal/summary.tsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+
+import GitHubUserAvatar from '@/js/components/githubUsers/avatar';
+import UserCards from '@/js/components/githubUsers/cards';
+import { ModalCard } from '@/js/components/tasks/capture';
+import { Dependency } from '@/js/store/projects/reducer';
+import { GitHubOrg, GitHubUser } from '@/js/store/user/reducer';
+
+interface Props {
+  name: string;
+  selectedOrg?: GitHubOrg;
+  selectedCollaborators: GitHubUser[];
+  selectedDeps: Dependency[];
+}
+
+const CreateProjectSummary = ({
+  name,
+  selectedOrg,
+  selectedCollaborators,
+  selectedDeps,
+}: Props) => {
+  const { t } = useTranslation();
+
+  return (
+    <form className="slds-form slds-p-around_large slds-m-bottom_medium">
+      <h3 className="slds-text-heading_small">
+        {t('Project Name: “{{project_name}}”', {
+          project_name: name,
+        })}
+      </h3>
+      <div className="slds-m-bottom_medium">
+        <h3 className="slds-text-heading_small slds-m-bottom_x-small">
+          {t('GitHub Organization')}
+        </h3>
+        <div
+          className="slds-button
+            slds-size_full
+            slds-p-around_xx-small
+            collaborator-button
+            is-assigned"
+          title={selectedOrg?.name}
+        >
+          <GitHubUserAvatar org={selectedOrg} />
+          <span className="collaborator-username">{selectedOrg?.name}</span>
+        </div>
```

Comment:
> I'm not sure if this larger label, smaller value is the pattern we wanted to aim for (I realize this screenshot is showing the organization that has it's own different style, but I am only referring to the pattern here). Not sure it even makes sense since it is in a heading, but it could be achieved with a span that controls text size.

@stacyk Yeah, I initially played with this inline like the mockup, then switched to the current implementation, which follows the pattern used in "Assign Developer" and "Assign Tester" modals when there is already someone assigned. Personally I like this better than having one item inline with a `:` and the rest as headers, but I'm open to changing if any of you really prefer the inline version.

![Screen Shot 2022-05-26 at 12 56 57 PM](https://user-images.githubusercontent.com/552316/170537427-bfc5e837-b8ec-401f-b442-8dfb7f9941ba.jpg)

Reply:
So something like this:

Project Name:
“Test Mechanics”
 
GitHub Organization:
jerivastest
 

Project Collaborators:
First Lastname
First Lastname


![Screen Shot 2022-05-26 at 12 11 36 PM](https://user-images.githubusercontent.com/1581694/170539710-ed52210d-1e99-4998-95bf-1913ff1e8ef8.jpg)


Instead of:

Project Name: “Test Mechanics”
 
GitHub Organization:
jerivastest
 

Project Collaborators:
First Lastname
First Lastname

 
![Screen Shot 2022-05-26 at 12 05 26 PM](https://user-images.githubusercontent.com/1581694/170538793-e4a5de49-4e0d-408e-a03c-57e7e0cf037a.jpg)

Reply:
@stacyk What do you prefer? I don't mind the name inline as much, I think because the org has an avatar and we already have a pattern for displaying that. But I really don't feel strongly on this, I'm happy to go with your recommendation.

Reply:
@jgerigmeyer I don't have a strong preference. I like the stacked version for consistency but inline for saving space in a modal that may have a  lot of collaborators so I suppose I would lean in that direction.

Reply:
Ok, I think I'll leave it as-is then?